app/assets/javascripts/admin/modules/document-history.js

```diff
@@ -0,0 +1,45 @@
+window.GOVUK = window.GOVUK || {}
+window.GOVUK.Modules = window.GOVUK.Modules || {};
+
+(function (Modules) {
+  function DocumentHistory (module) {
+    this.module = module
+  }
+
+  DocumentHistory.prototype.init = function () {
+    this.setupEventListeners()
+  }
+
+  DocumentHistory.prototype.setupEventListeners = function () {
+    var documentHistoryTab = document.querySelector('#history_tab')
+
+    document.addEventListener('click', function (e) {
+      var newerLink = document.querySelector('.app-component-document-history-tab__newer-pagination-link')
+      var olderLink = document.querySelector('.app-component-document-history-tab__older-pagination-link')
+
+      if (newerLink && newerLink.contains(event.target)) {
+        e.preventDefault()
+        // eslint-disable-next-line
+        Rails.ajax({
```

Comment:
The linter here complains that

```
standardx: Use JavaScript Standard Style (tweaked by standardx) (https://github.com/standard/standardx)
  /govuk/whitehall/app/assets/javascripts/admin/modules/document-history.js:22:9: 'Rails' is not defined.
  /govuk/whitehall/app/assets/javascripts/admin/modules/document-history.js:31:9: 'Rails' is not defined.
```
My understanding from googling is that when you use sprockets all that is required here is to add 

`//= require rails-ujs` to the `application.js` file.

any thoughts?